packages/shared/src/graphql/feedSettings.ts

```diff
@@ -98,6 +98,23 @@ export const FEED_SETTINGS_QUERY = gql`
   }
 `;
 
+export const FEED_FILTERS_FROM_REGISTRATION = gql`
+  mutation AddFiltersToFeed(
+    $filters: FiltersInput!
+    $settings: [FeedAdvancedSettingsInput]!
+  ) {
+    addFiltersToFeed(filters: $filters) {
+      id
+    }
+    updateFeedAdvancedSettings(settings: $settings) {
+      id
+    }
+    updateUserAlerts(data: { filter: false }) {
+      filter
+    }
```

Comment:
Wow nice!!

Is it still correct to do this manually in the sidebar? 
Or should we maybe consider later on refactoring to something similar?

Reply:
I am planning to refactor to apply it on users having the experiment disabled as we still need support the case outside experiment.